website/routes.py

```diff
@@ -882,6 +882,24 @@ def make_url_map(app):
             project_views.register.node_registration_embargo_disapprove,
             OsfWebRenderer('error.mako', trust=False)
         ),
+        Rule(
```

Comment:
I don't remember that discussion and prefer the query parameter. I'll think on it to see if I can recall why that would be problematic.